notebooks/work-in-progress/VCERARE.ipynb

```diff
@@ -0,0 +1,288 @@
+   "metadata": {},
+   "source": [
+    "If you want or need to alter the layout of the data in order to feed it into your model, are interested in a particular subset of the data, want to connect it to geospatial data, or want to view / run analysis on all the data at once, the processed PUDL Parquet files are the way to go! For context, the row limit in Excel is a little more than 1 million rows. The processed PUDL table contains 136,437,000 rows. Below, we'll show you how to wrangle this dataset without blowing up your computer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c399ea58-aa9b-45e1-9f0b-57adbc8d949e",
+   "metadata": {},
+   "source": [
+    "## Accessing RARE Parquet files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "77ef0416-3f43-48f2-b71a-76a383006f2c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import io\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import random\n",
+    "import sys\n",
+    "import pudl\n",
+    "pudl_paths = pudl.workspace.setup.PudlPaths()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "49394cc9-116b-4fee-a445-02afcc3215b1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rare_df = pd.read_parquet(\n",
+    "    pudl_paths.pudl_output / \"parquet/out_vceregen__hourly_available_capacity_factor.parquet\"\n",
+    ")\n",
+    "rare_df.sample(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d9553bd-3c69-4a70-b9e2-11dccd6cbfa5",
+   "metadata": {},
+   "source": [
+    "### Filter the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e87d1575-008c-4df9-a394-aaa3c875fde8",
+   "metadata": {},
+   "source": [
+    "You can use this format to filter the data you'd like to access. Here are some examples: \n",
+    "\n",
+    "```\n",
+    "# Filter by single year\n",
+    "filtered_df = rare_df[rare_df[\"report_year\"]==2023]\n",
+    "\n",
+    "# Filter by multiple years\n",
+    "filtered_df = rare_df[rare_df[\"report_year\"].isin([2019, 2020])]\n",
+    "\n",
+    "# Filter by state\n",
+    "filtered_df = rare_df[rare_df[\"state\"]==\"TX\"]\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "82d0cc7c-3b84-4dc8-b07d-4f2f6de8df79",
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "filtered_df = rare_df[rare_df[\"state\"]==\"RI\"]\n",
+    "filtered_df.head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e5c6c8d-8a8e-48be-918d-6a9d1e14741f",
+   "metadata": {},
+   "source": [
+    "### Visualize the data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cc41d9ee-3f6b-4da6-8450-6f649bef2744",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# random county_id generator:\n",
+    "random_county = random.choice(rare_df[\"county_id_fips\"])"
```

Comment:
Ideally here, "Select a random county by default, or substitute in your own 5-digit FIPS ID". I think called the variable `county_fips_id` instead of `random_county` will be better here.